### PR TITLE
Adding signal monitors to signal comms

### PR DESCRIPTION
**Pull request by Charlie-Henry**
## Associated issues
Adds new device IPs to the list of things to check.
https://github.com/cityofaustin/atd-data-tech/issues/9471

## Associated repo
https://github.com/cityofaustin/atd-signal-comms

## Testing

**Steps to test:**
Since these devices live behind the firewall, there is no way to test this code locally on your machine unless you have been whitelisted to the signal network (I have not). 

I do have an older [prefect deployment](https://app.prefect.cloud/account/62be21ad-87d4-4858-bc47-a49e1fd29df7/workspace/4e909dab-3495-4e26-9403-9b633af2c970/deployments/deployment/16536f09-55b2-4c6d-bd3f-4abc441412f7) that tests pinging these devices from data03 with the production docker container. Does not test the Socrata part, though.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_signal_comms.py [hunk 1]
```diff
@@ -155,7 +168,7 @@ def get_start_date(**context):
         command=f"python atd-signal-comms/socrata_pub.py detector --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
         environment=env_vars,
         tty=True,
-        force_pull=True,
+        force_pull=False,
```
Comment (Charlie-Henry) [inline dags/atd_signal_comms.py:171]: These extra `force_pull`s are unnecessary after the first one, I believe.

File: dags/atd_signal_comms.py [hunk 2]
```diff
@@ -61,16 +61,17 @@
...
 )
 def get_start_date(**context):
     """Get the --start date argument. Returns either the prev start date or today
     if the DAG has no successful run history"""
     from pendulum import now
-    prev_start_date = context.get("prev_start_date_success") or now()
-    return prev_start_date.strftime('%Y-%m-%d')
 
+    prev_start_date = context.get("prev_start_date_success") or now()
+    return prev_start_date.strftime("%Y-%m-%d")
```
Comment (Charlie-Henry) [inline dags/atd_signal_comms.py:74]: Black styling 🪄  here

**Review by johnclary — APPROVED** [review]
looks great! thanks for ditching those redundant force pulls 🧲 

🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Review by chiaberry — APPROVED** [review]
looks good

File: dags/atd_signal_comms.py (hunk 1 above)
Comment (frankhereford) [reply dags/atd_signal_comms.py:171]: Chia prompted me to adopt this pattern in a PR i have open Charlie - this is super smart. Thanks for this!

**Review by frankhereford — APPROVED** [review]
With the dependency on the signals network to really test this, I'm all in on your previous testing and say YOLO! 🚢

**Pull request merged**